src/helpers/group/group-helper.js

```diff
@@ -36,8 +36,8 @@ export async function addGroup(data) {
   return ret;
 }
 
-export async function removeGroup(groupId) {
-  return await groupApi.deleteGroup(groupId);
+export async function removeGroups(uuids) {
+  return await uuids.map((uuid) => groupApi.deleteGroup(uuid));
```

Comment:
You should wait for all deletes, that will also fix the problem with refreshing
```JS
return await Promise.all( uuids.map((uuid) => groupApi.deleteGroup(uuid)));
```

Reply:
Thanks, that fixed it! Trying to look into the tests now